tests/acceptance/21_methods/action_ifelapsed_respected.cf

```diff
@@ -0,0 +1,24 @@
+body common control
+{
+      inputs => { "../default.cf.sub" };
+      bundlesequence  => { default("$(this.promise_filename)") };
+      version => "1.0";
+}
+
+bundle agent test
+{
+  meta:
+    "description" string => "Test that methods are terminated as expected when ifelapsed is set";
+
+    # The subtest uses /bin/sleep so the test needs work on windows.
+    "test_soft_fail"
+      string => "any",
+      meta => { "CFE-2444" };
+}
+
+bundle agent check
+{
+  methods:
+    "" usebundle => dcs_passif_output(".*", ".*DID NOT TIMEOUT.*", "$(sys.cf_agent) -Kf $(this.promise_filename).sub", $(this.promise_filename));
```

Comment:
Not sure if this will work as the promise gets expired by the agent that runs afterwards, IIRC. So I think the next agent would kill the entire run of the current agent.